regions/io/core.py

```diff
@@ -3,19 +3,56 @@
 from __future__ import absolute_import, division, print_function
 from collections import OrderedDict
 from warnings import warn
+import copy
+import string
 
 from astropy import units as u
 from astropy import coordinates
-from astropy.coordinates import BaseCoordinateFrame
+from astropy.coordinates import BaseCoordinateFrame, Angle
 from astropy import log
 
-__all__ = ['ShapeList', 'Shape']
-
 from .. import shapes
-from ..core import PixCoord
+from ..core import PixCoord, SkyRegion, RegionMeta, RegionVisual
 from .ds9.core import DS9RegionParserWarning, DS9RegionParserError
 from .crtf.core import CRTFRegionParserWarning, CRTFRegionParserError
 
+__all__ = ['ShapeList', 'Shape', 'to_shape_list', 'to_crtf_meta', 'to_ds9_meta']
+
+regions_attributes = dict(circle=['center', 'radius'],
+                          ellipse=['center', 'width', 'height', 'angle'],
+                          rectangle=['center', 'width', 'height', 'angle'],
+                          polygon=['vertices'],
+                          annulus=['center', 'inner_radius', 'outer_radius'],
+                          line=['start', 'end'],
+                          point=['center']
+                          )
+
+# This helps to map the region names in the respective format to the ones available in this package
+reg_mapping = {'DS9': {x: x for x in regions_attributes.keys()},
+               'CRTF': {x: x for x in regions_attributes.keys()}}
+reg_mapping['DS9']['box'] = 'rectangle'
+reg_mapping['CRTF']['rotbox'] = 'rectangle'
+reg_mapping['CRTF']['poly'] = 'polygon'
+
+
+# valid astropy coordinate frames in their respective formats.
+valid_coordsys = {'DS9': ['image', 'physical', 'fk4', 'fk5', 'icrs', 'galactic',
+                          'geocentrictrueecliptic', 'wcs'],
+                  'CRTF': ['image', 'fk5', 'fk4', 'galactic', 'geocentrictrueecliptic', 'supergalactic', 'icrs']
```

Comment:
The line above exceeds 80 characters and could be wrapped

Reply:
Many of my lines cross the 80 limit(I wrap at 120), it seems ugly for some statements
Should I need to strictly follow 80 ?

Reply:
You do not have to strictly follow 80, but for lines like this, which naturally split at commas, you should.

Reply:
ok..!